nautobot/extras/tests/test_models.py

```diff
@@ -121,8 +123,8 @@ def setUp(self):
         self.platform = Platform.objects.create(name="Platform")
         self.tenantgroup = TenantGroup.objects.create(name="Tenant Group")
         self.tenant = Tenant.objects.create(name="Tenant", group=self.tenantgroup)
-        self.tag = Tag.objects.create(name="Tag", slug="tag")
-        self.tag2 = Tag.objects.create(name="Tag2", slug="tag2")
+        self.tag = Tag.objects.get_for_model(Device)[0]
+        self.tag2 = Tag.objects.get_for_model(Device)[1]
```

Comment:
```suggestion
        device_tags = Tag.object.get_for_model(Device)
        self.tag = device_tags[0]
        self.tag2 = device_tags[1]
```

Curious of the performance improvement...

Reply:
How about:

```suggestion
        self.tag, self.tag2 = Tag.objects.get_for_model(Device)[:2]
```